basic-services/ReadPropertyResult.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using Newtonsoft.Json.Linq;
+
+
+namespace JohnsonControls.Metasys.BasicServices
+{
+
+
+    public struct ReadPropertyResult
+    {
+        // Follow rules for stringValue in MSSDA Bulletin, with the following
+        // addendum. The string value for an Array will be "Array".
+        public string StringValue { private set; get; }
+        
+        // Follow rules for rawValue in MSSDA Bulletin, with the following 
+        // addendum. The numeric value for an Array will be 0
+        // Enums will also be 0 (since we don't expose their numbers, we just expose them as strings)
+        public double NumericValue { private set; get; }
+        
+        // This is new. MSSDA required you to read just one element of an array.
+        // We will support reading the entire array (asusming it's an array of primitives)
+        // This will be null for non-array data types.
```

Comment:
Alternatively, we could set ArrayValue to a single item array with just the one element in it.